runner-charts/deployment/values.yaml

```diff
@@ -20,6 +20,17 @@ cpuLimit: "500m"
 
 privileged: false
 
+# Enable cluster wide proxy
+# https://docs.openshift.com/container-platform/4.6/networking/enable-cluster-wide-proxy.html
+clusterProxy: false
+https_proxy: ""
```

Comment:
Helm values should be camel case. https://helm.sh/docs/chart_best_practices/values/#naming-conventions

Reply:
Thanks for this. 
I've never followed naming conventions. I am finding myself reviewing the 10+ charts I've written 🤣